ui/settings.js

```diff
@@ -127,6 +127,21 @@ function create_settings_ui() {
     $(str).appendTo($('.panel-body'));
 }
 
+function get_icon_map() {
+    fs = require('fs');
+    let icon_dir = 'node_modules/cryptocurrency-icons/svg/color/';
+    let files = fs.readdirSync(icon_dir)
```

Comment:
This is not correct. It's not a map. You are creating an array here. And then you will have to search the entire array which is really bad for performance. What I mean is to create a map like that:

```python
{ 
    'ETH': 'path/to/ethicon.png', 
    'BTC': 'path/to/btcicon.png'
}
```